tsdb/importer/openmetrics/openmetrics.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openmetrics
+
+import (
+	"bufio"
+	"io"
+
+	"github.com/prometheus/prometheus/pkg/textparse"
+)
+
+// Content Type for the Open Metrics Parser.
+const contentType = "application/openmetrics-text;"
```

Comment:
This isn't quite right, there should be a version in here. Let's not risk users blindly copying and pasting this incorrect value elsewhere.

Reply:
I am sorry I don't understand your comment. Could you please explain it in some detail.

Reply:
Use a full proper content-type for OM, like https://github.com/prometheus/client_python/blob/master/prometheus_client/openmetrics/exposition.py#L7

Reply:
Okay thanks for clarification